src/combine-results.ts

```diff
@@ -59,7 +59,7 @@ async function run(): Promise<void> {
         const output = await getExecOutput("wc", ["-l", csvDest]); // TODO: preferably we would do this during results interpretation
         const results = parseInt(output.stdout.trim()) - 2;
 
-        if (results > 0) {
+        if (results >= 0) {
```

Comment:
I'm confused - does it return `-1` for no results. The case at the bottom should would not fire anymore otherwise:

```
return `| ${repoName} | _No results_ |`;

If that's actually the case, should line 70 change to:

```
return `| ${repoName} | [${results + 1} result(s)](${comment.data.html_url}) |`;
```
```